src/controllers/account.controller.js

```diff
@@ -13,12 +14,17 @@ export default {
   * @param {string} res
   * @returns {object} returns all accounts
   */
-  index(req, res) {
-    const accounts = AccountModel.all();
-
-    if (accounts) { return res.json({ message: 'Operation successful', status: 200, accounts }); }
-
-    return res.json({ status: 400, message: 'No accounts created' });
+  async index(req, res) {
```

Comment:
Why `index`, give the function a descriptive name

Reply:
Noted.